sorl/thumbnail/templatetags/sorl_thumbnail.py

```diff
@@ -0,0 +1,8 @@
+"""
+This allows usage of sorl-thumbnail in templates
+by {% load sorl_thumbnail %} instead of traditional
+{% load thumbnail %}. It's specifically useful in projects
+that do make use of multiple thumbnailer libraries (for
+instance `easy-thumbnails` alongside `sorl-thumbnail`).
+"""
+from .thumbnail import *
```

Comment:
Would you mind adding `  # noqa` at the end of this line to avoid editors/linters complaining about the asterisk import (this is a valid user case for an asterisk import).